@types/jsb.d.ts

```diff
@@ -566,109 +576,143 @@ declare namespace jsb {
+             * 未知状态
+             */
+            UNSPECIFIED_STATE = 0,
+        }
     /**
      * @en Represents an in-app billing purchase.
      * @zh 代表应用内billing购买。
      */
-    export interface Purchase {
+    export class Purchase {
+        static PurchaseState: typeof jsb.PurchaseState;
         /**
          * @en One of PurchaseState indicating the state of the purchase.
          * @zh PurchaseState表示购买状态的其中一个值。
          */
-        readonly purchaseState: number;
+        getPurchaseState(): number;
         /**
          * @en The time the product was purchased, in milliseconds since the epoch (Jan 1, 1970).
          * @zh 产品购买的时间，以纪元（1970 年 1 月 1 日）以来的毫秒数表示。
          */
-        readonly purchaseTime: number;
+        getPurchaseTime(): number;
         /**
          * @en Indicates whether the purchase has been acknowledged.
          * @zh 表示是否已确认购买。
          */
-        readonly isAcknowledged: number;
+        isAcknowledged(): number;
         /**
          * @en Indicates whether the subscription renews automatically.
          * @zh 指示订阅是否自动续订。
          */
-        readonly isAutoRenewing: number;
+        isAutoRenewing(): number;
         /**
          * @en Hash code
          * @zh hash值
          */
-        readonly hashCode: number;
+        hashCode(): number;
 
         /**
          * @en The quantity of the purchased product.
          * @zh 购买产品的数量。
          */
-        readonly quantity: number;
+        getQuantity(): number;
         /**
          * @en The payload specified when the purchase was acknowledged or consumed.
          * @zh 确认或消费购买时指定的有效负载。
          */
-        readonly developerPayload: string;
+        getDeveloperPayload(): string;
         /**
          * @en Returns a unique order identifier for the transaction.
          * @zh 交易的唯一订单标识符。
          */
-        readonly orderId: string;
+        getOrderId(): string;
         /**
          * @en Returns a String in JSON format that contains details about the purchase order.
          * @zh 包含有关采购订单详细信息的 JSON 格式的字符串。
          */
-        readonly originalJson: string;
+        getOriginalJson(): string;
         /**
          * @en The application package from which the purchase originated.
          * @zh 购买来源的应用程序包。
          */
-        readonly packageName: string;
+        getPackageName(): string;
         /**
          * @en A token that uniquely identifies a purchase for a given item and user pair.
          * @zh 唯一标识给定商品和用户对的购买的令牌。
          */
-        readonly purchaseToken: string;
+        getPurchaseToken(): string;
         /**
          * @en String containing the signature of the purchase data that was signed with the private key of the developer.
          * @zh 包含使用开发者私钥签名的购买数据签名的字符串。
          */
-        readonly signature: string;
+        getSignature(): string;
         /**
          * @en To string
          * @zh 转换成字符串
          */
-        readonly toStr: string;
+        toStr(): string;
```

Comment:
How about using `toString()` to keep the same name which in JS and Java?

Reply:
fixed